azure/Guidelines.md

```diff
@@ -0,0 +1,242 @@
+# Microsoft Azure REST API Guidelines
+
+## History
+
+| Date | Version | Notes |
+| 2020-Mar-31 | v3.1 | 1st public release of the Azure REST API Guidelines|
+
+## Introduction
+
+The Azure REST API guidelines are an extension of the [Microsoft REST API guidelines][1]. Readers of this document are assumed to be also reading the [Microsoft REST API guidelines][1] and be familiar with them.  Azure guidance is a superset of the Microsoft API guidelines and services should follow them *except* where this document outlines specific differences or exceptions to those guidelines. This document does contain additional Azure-specific guidance and additional details.
+
+#### Asynchronous operations
```

Comment:
This feels like a section that should move outside the Introduction.

Reply:
Also, small nit: would it make sense to name this section the same way it appears in the MS REST API Guidelines, i.e. Long running operations?  It would be nice for readers to be able to map the ideas in these guidelines back to the MS guidelines with ease.

Reply:
Agreed.  I'll promote it to its own section.